src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -1,121 +1,17 @@
 package juja.sqlcmd;
 
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
-import java.sql.Statement;
 
-public class DatabaseManager {
-    private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
+public interface DatabaseManager {
```

Comment:
Please add method for update operation. 
Yes, we are going to change parameters for this method,  but without this case we will forget about it

Reply:
OK. Will be done